pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -687,13 +680,7 @@ func (cb *ClusterBuilder) shouldH2Upgrade(clusterName string, direction model.Tr
 		return false
 	}
 
-	if !h2UpgradeMap[upgradeTuple{mesh.H2UpgradePolicy, override}] {
-		log.Debugf("Not upgrading cluster: %v (%v %v)", clusterName, mesh.H2UpgradePolicy, override)
-		return false
-	}
-
-	log.Debugf("Upgrading cluster: %v (%v %v)", clusterName, mesh.H2UpgradePolicy, override)
-	return true
+	return mesh.H2UpgradePolicy == meshconfig.MeshConfig_UPGRADE
```

Comment:
Can you add some comments here and retain the debug log line which says upgrading the cluster as per default mesh config policy?

Reply:
I think the log is redundant here,  this is mesh global H2UpgradePolicy.

Reply:
IC